lkvt.go

```diff
@@ -261,23 +319,43 @@ func (conf *config) execute(c int, ran []uint32, wg *sync.WaitGroup) {
 			randVal:   toByteArray(ran[i]),
 			count:     uint32((*conf.amount / *conf.concurrency)*c + i),
 		}
+		etcd := "etcd"
 		if float64(i) < float64(*conf.amount / *conf.concurrency)*(*conf.putPercentage) {
 			kv.createKV(0)
-			putTimer := time.Now()
-			kv.sendClientPut()
-			// end put timer
-			stopPutTime := time.Since(putTimer)
-			// add time to put array
-			conf.putTimes = append(conf.putTimes, stopPutTime)
+			if *conf.database == etcd {
+				putTimer := time.Now()
```

Comment:
timer start and stop should not be inside the conditional.  Instead, let's move it to a new method, `op_execute()`, which does something like:
```
timer = time.Now()
switch (op_type)
case PUT: kv.lkvtOp()
                  time_array = &conf.PutTimes
case GET: kv.lkvtOp()
                  time_array = &conf.GetTimes
end_time = time.Since(timer)
time_array = append...
```

Now, lkvOp should route the request to the proper backend method etcd or niova, such as kv.etcdOp() and kv.niovaOp().  these Op methods should then call the put or get functions accordingly.